### PR TITLE
Add handling wikis from a Gitlab Group

### PR DESCRIPTION
**Pull request by Tan-JamickoJohn**
**Description:**
The ability to handle wikis in GitHelp before only assumes that the current project is the GitHelp repo.
Now, we can clone and pull from a wiki of a GitLab group, and then update the current project's wiki with the Wiki from the GitLab group.

**Validation Steps**
- Switch to this version of Githelp
- Pick any project and use `ghCO` to clone that.
- Run `ghCW` to have a local copy of the current project's wiki
- Run `ghCWFGG <gitlab_group_name>` to clone the gitlab group's wiki
- Running `ghCWFGG <gitlab_group_name>` should return a message stating that `ghPWFGG` must be ran
- Try to make a change on the gitlab group's wiki then run `ghPWFGG <gitlab_group_name> `, this  pull in the change you just made
- Run `ghUWFGG <gitlab_group_name>` to push the GitLab group's wiki to your current project

### CODE REVIEW

**Review by flywheelms — APPROVED** [review]

File: GitHelp/ghCloneWiki.sh [hunk 1]
```diff
@@ -4,24 +4,18 @@
...
 
 if [ -d $REPO_ROOT ]; then
     printf "\n$REPO_ROOT already exists.\nOperation canceled.\n\n"
-    printf "Use gitPW to get the latest wiki files.\n\n"
+    printf "Use ghPW to get the latest wiki files.\n\n"
     exit 1
 fi
 
-eval UPSTREAM="${GIT_HOST_URL}/${UPSTREAM_OWNER}/${REPO_NAME}.git"
+eval UPSTREAM="${GIT_HOST_URL}.wiki.git"
```
Comment (joanna-enzo) [inline GitHelp/ghCloneWiki.sh:18]: What is the purpose of the script? My understanding, based on the line at the top is that the `ghCW` command should clone the wiki for the upstream repository. So if I've checked out my fork of the GitHelp repository, when I run `ghCW`, it should clone the wiki associated with the upstream GitHelp repository, not the wiki that could be attached to my fork of GitHelp. (I think the 'Assumption' on line 5 refers to which repository (GitHelp, MyFlutterApp, etc. is to be referenced and not whether we clone fork or upstream.)

The current behavior of the script is to clone the wiki associated with a fork. When I ran `ghCW` while in the directory for my fork of GitHelp, I was prompted to `Clone WIKI from https://github.com/joanna-enzo/GitHelp.wiki.git`. That wiki however is empty. Wikis do not forked from the upstream repository as the code does.

If my assumption of the expected behavior is correct, then you can achieve it by replacing line 18 with the following two:

```
UPSTREAM_PROJECT=`git config --get remote.upstream.url | sed 's/\.git//'`
eval UPSTREAM="${UPSTREAM_PROJECT}.wiki.git"
```

**Review by joanna-enzo — COMMENTED** [review]
I validated all the wiki related commands that were modified, and I encountered some unexpected behavior. Please see the individual comments for suggestions on how to resolve these.

The `ghUWFGG` command purpose/implementation is unclear and does not seem to be consistent with the PR description. Let's chat about this one to so we're on the same page on the expected behavior.

**Comment by jamicko-tan-enzo** [comment]
From my last discussion with @joanna-enzo  here are the updates
- `ghCW` : update to clone upstream repo
- `ghUWFGG` : change functionality to update the wiki of the Gitlab Group
- fixed alias and descriptions
- used absolute pathing instead of the relative path to a wiki

File: GitHelp/ghCloneWiki.sh (hunk 1 above)
Comment (joanna-enzo) [reply GitHelp/ghCloneWiki.sh]: Re-validated after change to confirm the `ghCW` works as expected.

**Review by joanna-enzo — APPROVED** [review]
Reviewed all the updates and re-validated the commands. Looks great!

**Pull request merged**